specs/breadcrumb.md

```diff
@@ -0,0 +1,139 @@
+He clicks on `Kitchen and Dining` to go back to the category of kitchen items, selects `Coffee and Espresso` finds the best one that suits his needs. If James wants to continue looking for another item for his kitchen, or other things for his home, he just needs to click on any of the links in the breadcrumb trail to continue his search.
+
+### Prior Art/Examples
+- [Ant Design](https://ant.design/components/breadcrumb/)
+- [Atlaskit](https://atlaskit.atlassian.com/packages/core/breadcrumbs)
+- [Carbon Design](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [FAST Breadcrumb (React)](https://www.npmjs.com/package/@microsoft/fast-components-react-msft)
+- [Lightning Design System](https://www.lightningdesignsystem.com/components/breadcrumbs/#site-main-content)
+- [Material UI](https://material-ui.com/components/breadcrumbs/)
+- [Office Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb)
+
+---
+
+### API
+
+*Component Name*
+- `fast-breadcrumb`
+
+*Properties*
+- `slottedBreadcrumbItems` - an HTMLElement[] used in the slotted directive.
+
+*Attribute*
+- aria-current - attribute set on the last breadcrumb item, if it has an `href` set. It will always be `aria-current="page"`.
+
+### Anatomy and Appearance
+
+```html
+<template
+    role="navigation"
+    aria-label="breadcrumb"
+>
+    <div role="list">
+        <slot ${slotted({ property: "slottedBreadcrumbItems", filter: elements() })}></slot>
+    </div>
+</template>
+```
+
+*Slot Names*
+- slotted directive, with a property called `slottedBreadcrumbItems`.
+
+---
+
+## Implementation
+
+```html
+<fast-breadcrumb>
+    <fast-breadcrumb-item name="Home" href="#"></fast-breadcrumb-item>
+    <fast-breadcrumb-item name="Template" href="#"></fast-breadcrumb-item>
+    <fast-breadcrumb-item name="Template Prices" href="#"></fast-breadcrumb-item>
+</fast-breadcrumb>
+```
+
+### Accessibility
+
+The breadcrumb should align to the interaction model provided by the [W3C](https://www.w3.org/TR/wai-aria-practices/#breadcrumb)
+- `aria-current` - W3 specs says when using `aria-current="page"` the element should be a link. `fast-breadcrumb` will set `aria-current="page"` to the last element if there is an href attribute.
+
+### Globalization
+
+The first and last element inside the breadcrumb will be reversed when switching in LTR/RTL.
+
+### Test Plan
+
+While testing is still TBD for our web components, I would expect this to align with the testing strategy and not require any additional test support.
+
+
+
+# Breadcrumb Item
+
+## Overview
+
+The `fast-breadcrumb-item` is placed inside the `fast-breadcrumb` component. It provides a slot with a default anchor. The component also provides a slot for a separator that defaults using a `/`.
+
+### API
+
+*Component Name*
+- `fast-breadcrumb-item`
+
+*Properties*
+- `showSeparator` - is a boolean to show and hide the separator.
```

Comment:
If I'm using fast-breadcrumb with these items, do I need to set this on every item if I want to change from the default (assuming the default is true, I want to hide it and set false)? Or is the intent with this being a property that we have a method of hiding all nested fast-breadcrumb-items separators if we want?

Reply:
Yes, `showSeparator` is true by default. We can create a method that will hide all separator, currently, my thought was to only hide the separator on the last item. In Breadcrumb, I will use the `instanceof` BreadcrumbItem, and on the last item, set `showSeparator` to false. 
You also brought up a point on changing separator on each item from default. I will have to come up with a way to only change the separator once and not each item.

Reply:
Yes, `showSeparator` is defaulted true. In Breadcrumb, I would use `instanceof` BreadcrumbItem and set `showSeparator` on the last item to false. Since the separator visually represents a breadcrumb trail, I did not think to hide all of them.
To change the default of the separator, I will have to come up with a way to change it once, and not have to change it on all the items.